### DISCUSSION TITLE
DigitalCriminal, are you alright?

### DISCUSSION THREAD

**Discussion by A-Random-Mf** [DIGITALCRIMINALS/OnlyFans#408] [General]
Been awhile since I saw a message of yours. Just checking

**Comment by DIGITALCRIMINAL** [comment]
I'm still active, just busy. I'm still active on Twitter though.

**Reply by A-Random-Mf** [reply]
Oh wonderful, good to hear from you

**Comment by J4k-The-Reaper** [comment]
Hey man. I completely understand you being busy and prolly don't have as much time to mess around with the scraper now that the whole pandemic is pretty much over-finally-.....but any reason there won't be possible update or "patch"work/  fixes to the scraper?

**Reply by DIGITALCRIMINAL** [reply]
Clocked in nearly 1000 hours on Lost Ark since release, that's what's been taking up my time.

**Reply by J4k-The-Reaper** [reply]
Damn! That is a good reason! I totally understand. This generation....video games are life! Haha.
